simulator/infra/replacement/cache_replacement.cpp

```diff
@@ -4,6 +4,7 @@
  * @author Oleg Ladin, Denis Los, Andrey Agrachev
  */
 
+#include "infra/cache/cache_tag_array.h"
```

Comment:
`#include <sparsehash/dense_hash_map.h>`

Reply:
?

Reply:
@agrachiv99 You keep missing this

Reply:
.

Reply:
I am sorry - completely missed this